python/src/scipp/table_html/formatting_html.py

```diff
@@ -292,53 +287,46 @@ def _format_common(is_index):
     return cssclass_idx, attrs_id, attrs_icon, data_id, data_icon
 
 
-def _variable_format(name, dims_str, dtype, unit, preview, variances_preview,
-                     attrs_ul, disabled, data_repr, is_index, has_attrs):
+def summarize_variable(name,
```

Comment:
This is a bit of bike-sheding, but summarize doesn't really mean the same as format
Could we use `format_variable` or `variable_to_html` or something similar?

Reply:
I think you are misreading the diff, I have not renamed this function, but moved its content into `summarize_variable`. There are many similarely named functions in the file, so this is consistent. Then, there are several lower-level functions named `_format_*`, so you would need to come up with a completely new naming scheme, which is outside the scope of this PR.

Reply:
:+1: the diff looks like the name has been changed rather than the implementation merged hence my confusion